.gitignore

```diff
@@ -6,43 +6,23 @@ coverage
 npm-debug.log
 .idea
 *.iml
+components/.DS_Store
```

Comment:
you can just make this `.DS_Store` and then it will ignore it everywhere.

Reply:
ok thanks updated